scripts/README.md

```diff
@@ -0,0 +1,8 @@
+# Prepare csv dump of Jira tickets adjusted for embedding
+```commandline
+uv run csv_to_json.py process_jira_tickets.csv jira_config.ini output.jsonl
```

Comment:
What do you think about providing a link here to installing UV as a prereq for this? 

(I do see the alternative below to using python directly, given that you mentioned uv is quite convenient and the user may not have it installed, this link might help them get setup quickly)

Reply:
Python should also work.

Reply:
added